flash/core/utilities/imports.py

```diff
@@ -20,6 +20,8 @@
 from typing import Callable, List, Tuple, Union
```

Comment:
```suggestion
from typing import List, Tuple, Union
```

`Callable` is unused now :)